packages/truffle-core/lib/services/google-analytics.js

```diff
@@ -0,0 +1,157 @@
+const Config = require("truffle-config");
+const userConfig = Config.getUserConfig();
+const ua = require('universal-analytics');
+const nanoid = require('nanoid');
+const inquirer = require("inquirer");
+
+const version = require("../version");
+
+//set truffleAnalyticsId depending on whether version is bundled
+let bundle_version;
+let appVersion;
+let truffleAnalyticsId;
+
+if (version.bundle) {
+  appVersion = "v " + version.bundle;
+  truffleAnalyticsId = "UA-83874933-6";
+} else {
+  appVersion = "(unbundled) " + "version " + version.core;
+  truffleAnalyticsId = "UA-83874933-7";
+}
+
+
+const analyticsInquiry = [
+  {
+    type : "list",
+    name : "analyticsInquiry",
+    message : "Would you like to enable analytics for your Truffle projects? Doing so will allow us to make sure Truffle is working as expected and address any bugs more efficiently.",
```

Comment:
The grammar is slightly weird near the end...maybe `Doing so will allow us to make sure Truffle is working as expected and help us address any bugs more efficiently.`